src/app.scss

```diff
@@ -1,44 +1,15 @@
 #root {
-  height: 100%;
-  min-height: 850px;  
```

Comment:
avoid hard-coding height, since it will make it hard to make the website responsive to smaller devices.